robocopkaka/hospital
# #161097262 Patients can book appointments

#### What does this PR do?
Allow patients to make appointments
#### Description of Task to be completed?
* Allow patients select the specialization they want to book their appointments with
#### How should this be manually tested?
* Login/Signup as a patient
* Click on the `Make an appointment` link on the navbar
* Enter the appointment date
* Select the specialization
* Click `Submit`
* You should be redirected to the home page and see a flash message telling you that your appointment has been created
#### What are the relevant pivotal tracker stories?
[#161097262](https://www.pivotaltracker.com/story/show/161097262)
#### Any background context you want to add?
N/A
#### Screenshots

![screen shot 2018-10-15 at 2 07 27 pm](https://user-images.githubusercontent.com/11176000/46952948-4f548600-d084-11e8-840d-77362af5ca03.png)
